src/modules/acknowledge_challenge.json

```diff
@@ -21,10 +21,14 @@
       "tags": [
         {
           "name": "event",
-          "x-provides": ["xrn:firebolt:capability:usergrant:acknowledgechallenge"],
           "x-response": {
             "$ref": "#/components/schemas/GrantResultProviderResponse"
           }
+        },
+        {
+          "name": "capabilities",
+          "x-provides": "xrn:firebolt:capability:usergrant:acknowledgechallenge",
+          "x-allow-focus": true
```

Comment:
we can discuss more, but...

I'm trying to codify which extensions go on which tags. `x-provides` is clearly a capabilities ext. Same with `x-allow-focus`.

`x-response` _feels_ like it could be used for non-capability app interfaces... like what if every app had to handle a basic getter with some value to be considered a valid app? That _could_ be a provider, but that seems overkill. 

Anyway, I just kind of picked `event` as the tag for responses, because only events can trigger a response.

Reply:
I'm currently writing a meta-schema to enforce all of our Firebolty OpenRPC assumptions. once it's done, we can debate all of the rules that have effectively been made up.

I suggest we decide the final home for this tag when I make that PR (hopefully next week)